### DISCUSSION TITLE
Awesome blogs about APISIX

### DISCUSSION THREAD

**Discussion by moonming** [apache/apisix#8837] [Show and tell]
Welcome to post some awesome blogs related to Apache APISIX in this discussion

**Comment by juzhiyuan** [comment]
- Title: Why Is Apache APISIX the Best API Gateway?
- Link: https://api7.ai/blog/why-is-apache-apisix-the-best-api-gateway

-----

- Title: Powering AI Capabilities with Apache APISIX and OpenAI API
- Link: https://api7.ai/blog/power-ai-capabilities-with-apache-apisix

**Comment by shreemaan-abhishek** [comment]
- Title: Why would you choose Apache APISIX instead of NGINX or Kong?
- Link: https://apisix.apache.org/blog/2021/06/28/why-we-need-apache-apisix/
